Automattic/wp-calypso
# Get Apps: Delete unused/redundant code in Get Apps component

A `jetpack/app-branding` feature flag was added in https://github.com/Automattic/wp-calypso/pull/6681 in order to conditionally replace mentions of the WordPress app with the Jetpack app. As the flag is now set to `true` across all environments, we can safely remove all code that mentions the WordPress app, as it is now redundant and never used.

This PR is one of multiple planned that will eventually allow us to remove the `jetpack/app-branding` feature flag.

## Proposed Changes

* All code that is specific to the WordPress app within the Get Apps component has been removed.

## Testing Instructions

* Navigate to _Me_ → _Get Apps_ with either this branch checked out locally or with [the live branch](https://github.com/Automattic/wp-calypso/pull/74263#issuecomment-1461868385) generated in the comments of this PR. 
* Verify that the page's design and functionality has not changed following the removal of code in this PR.
* Testing should take place across multiple screen sizes.

| Mobile card  | Desktop card |
| ------------- | ------------- |
| <img src="https://user-images.githubusercontent.com/2998162/224023329-1dce7e15-36f6-41f6-9a27-0eef63334e21.png" width="100%"> | <img src="https://user-images.githubusercontent.com/2998162/224023434-43080a43-4c96-49f3-ac0b-c7012d6f3d55.png" width="100%"> |

## Pre-merge Checklist
